content/docs/csidriver/installation/operator/operator_migration.md

```diff
@@ -30,7 +30,7 @@ description: >
       kubectl -n openshift-operators get CSIUnity/test-unity -o yaml
   ```
 2. Map and update the settings from the CR in step 1 to the relevant CSM Operator CR
-    - As the yaml content may differ, ensure the values held in the step 1 CR backup are present in the new CR before installing the new driver
+    - As the yaml content may differ, ensure the values held in the step 1 CR backup are present in the new CR before installing the new driver. CR Samples table provided above can be used to compare and map the differences in attributes between Dell CSI Operator and CSM Operator CR s
```

Comment:
At the end of the line "... and CSM Operator CRs"

Reply:
@rajendraindukuri : Please address above comment